### PR TITLE
Chore: Adapt Compose Gradle Plugin

### PR DESCRIPTION
**Pull request by b-re-w**
Hi! @sridhar-sp 

Here is a suggestive commit to adapt compose gradle plugin for this repo.
This will help to maintain compose library versions.

Please review this PR.

V/r

Thank you.

### CODE REVIEW

**Comment by sridhar-sp** [comment]
Hope this is tested locally, 
Will review and reply by tomorrow.

**Comment by IRACK000** [comment]
Thank you for the rapid response!

File: neumorphic/build.gradle [hunk 1]
```diff
@@ -58,7 +59,10 @@ def getArtifactName() {
 
 dependencies {
     api 'androidx.core:core-ktx:1.7.0'
-    api "androidx.compose.ui:ui:$compose_version"
+    api compose.ui
+    api compose.runtime
+    api compose.foundation
+    api compose.material
```
Comment (sridhar-sp) [inline neumorphic/build.gradle:65]: Compose runtime , foundation , and material dependency may not be needed here.

Comment (sridhar-sp) [reply neumorphic/build.gradle:65]: Merging this commit. if those dependency are redundant. I'll make a separate commit to remove those

**Review by sridhar-sp — APPROVED** [review]
LGTM

**Pull request merged**